airflow/utils/log/log_reader.py

```diff
@@ -54,6 +57,8 @@ def read_log_chunks(
         contain information about the task log which can enable you read logs to the
         end.
         """
+        if ti.operator == DummyOpeartor.__class__.__name__:
```

Comment:
Would prefer to avoid magic methods (+ typo in `DummyOperator`). Plus it should also account for the usage of ExternalTaskMarker.

```suggestion
        if isinstance(ti.operator, DummyOperator):
```

Reply:
@BasPH I would like to use isinstance but  ti.operator is just a string so isinstance does not work like this. I will fix the typo 🤦